.travis.yml

```diff
@@ -58,6 +61,13 @@ matrix:
     -  python: 3.6
        env: DISTRIB="scipy-dev"
        if: type = cron OR commit_message =~ /\[scipy-dev\]/
+    # this tests that scikit-learn builds and passes on 32 bit machines
+    # currently uses a centOS5 32 bit distribution from a docker image
+    # see build_tools/travis/install.sh for information
+    # Don't need to run tests on travis since everything will be done in the
+    # docker container
+    - env: DISTRIB="32bit" COVERAGE=false TEST_DOCSTRINGS=false SKIP_TESTS=true RUN_FLAKE8=false
```

Comment:
We do want to test docstrings too I think

Reply:
assumed resolved by @amueller 's comment (we're not going to test them in 32 bit)